monkey/infection_monkey/ransomware/utils.py

```diff
@@ -0,0 +1,96 @@
+import os
+from typing import List
+
+VALID_FILE_EXTENSIONS_FOR_ENCRYPTION = {
```

Comment:
Move to a separate file at least. There's no reason for someone to scroll through 60 lines of file extensions just to be able to get to the logic of file enumeration.